templates/zerver/help/create-a-stream.md

```diff
@@ -30,8 +30,8 @@ be no longer than 60 characters.
     in the stream overview.
 
 7. The **Stream accessibility** option controls the privacy of your stream. There are two options:
-    - **Anybody can join** - This option makes your stream **public** and accessible to all users.
-    - **People must be invited** -- This option makes your stream **private**.  Only users you invite will be able to access this stream.
+    * **Anybody can join** - This option makes your stream **public** and accessible to all users.
```

Comment:
If you're reviewing this doc, you could also fix line lengths to be <= 76 characters.